src/components/application_manager/test/policy_handler_test.cc

```diff
@@ -1995,6 +2013,7 @@ TEST_F(PolicyHandlerTest, GetAppIdForSending_ExpectReturnAnyIdButNone) {
       .WillRepeatedly(Return(app_in_none_id));
   EXPECT_CALL(*mock_app_in_none, hmi_level())
       .WillRepeatedly(Return(mobile_api::HMILevel::HMI_NONE));
+  EXPECT_CALL(*mock_app_in_none, device()).Times(0);
```

Comment:
@ValeriiMalkov  strange device

Reply:
@LuxoftAKutsan Fixed in commit cb0fa9a5bd7649dc7322d420554dc10fdaf4adff